stdlib/contrib/bonitoo-io/hex/hex.flux

```diff
@@ -1,7 +1,156 @@
+// Package hex provides functions that perform hexadecimal conversion
+// of `int`, `uint` or `bytes` values to and from `string` values.
+//
+// introduced: 0.131.0
 package hex
 
-
+// int converts a hexadecimal string to an integer.
+// tag: type-conversion
+//
+// ## Parameters
+//
+// - v: String to convert.
+//
+// ## Examples
+// ### Convert hexadecimal string to integer
+// ```
+// import "contrib/bonitoo-io/hex"
+//
+// hex.int(v: "4d2")
+//
+// // Returns 1234
+// ```
 builtin int : (v: string) => int
+
+// string converts a Flux basic type to a hexadecimal string.
+// tag: type-conversion
```

Comment:
these tags and other metadata must be the last lines of the comment or they will not be parsed.